sql/mysql/Positive-Technologies/MySqlLexer.g4

```diff
@@ -189,6 +190,7 @@ SET:                                 'SET';
 SEPARATOR:                           'SEPARATOR';
 SHOW:                                'SHOW';
 SIGNAL:                              'SIGNAL';
+SKIP_:                               'SKIP';
```

Comment:
Should it be added to [keywordsCanBeId](https://github.com/antlr/grammars-v4/blob/master/sql/mysql/Positive-Technologies/MySqlParser.g4#L2591)?